dcicutils/beanstalk_utils.py

```diff
@@ -233,9 +234,9 @@ def _compute_prd_env_for_project(project):
     magic_cname = CGAP_MAGIC_CNAME if project == 'cgap' else FF_MAGIC_CNAME
     client = boto3.client('elasticbeanstalk', region_name=REGION)
     res = describe_beanstalk_environments(client, ApplicationName="4dn-web")
-    logger.info(res)
+    # logger.info(res)
```

Comment:
I would just delete these lines altogether. They didn't serve much purpose before IMO and certainly don't now.

Reply:
Done.